src/Databases/MysqlDatabase.php

```diff
@@ -34,6 +34,13 @@ public function getDumpCommandLine($outputPath) {
     	if (array_key_exists('singleTransaction', $this->config) && $this->config['singleTransaction'] === true) {
     		$extras[] = '--single-transaction';
     	}
+        if (array_key_exists('ignoreTables', $this->config) && is_array($this->config['ignoreTables']) && count($this->config['ignoreTables'])) {
```

Comment:
Hi, thanks for the addition. Could you please extract this added code to a method?